src/Helmet.js

```diff
@@ -41,7 +41,16 @@ const getTitleFromPropsList = (propsList) => {
 };
 
 const getOnChangeClientState = (propsList) => {
-    return getInnermostProperty(propsList, "onChangeClientState") || () => {};
+    return getInnermostProperty(propsList, "onChangeClientState") ||(() => {});
+};
+
+const getHtmlAttributesFromPropsList = (propsList) => {
+    return propsList
+        .filter(props => !Object.is(typeof props[TAG_NAMES.HTML], "undefined"))
+        .map(props => props[TAG_NAMES.HTML])
+        .reduce((html, current) => {
+            return {...html, ...current};
```

Comment:
This works, we can keep this open-ended and let people police themselves I guess.  It would be too many tags to manage.